.husky/pre-merge-commit

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
+npm run check-types
```

Comment:
Checks types on merge commit.

Does it make more sense to **not**  run checks on merge commit, and  leave it to pre-push and CI?
- it won't be often that merge commits cause checks to fail, so letting them fail pre-push/CI is probably okay
- if this pre-merge-commit fails it could leave working tree in a potentially fiddly state

Reply:
Yes it's probably okay now that we can't push unclean trees